tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/subbuilder/LiteralAndNameBuilder.scala

```diff
@@ -16,7 +16,7 @@ trait LiteralAndNameBuilder {
   private val unsafeBuilder = new UnsafeLiteralAndNameBuilder
 
   /** {{{i : Int}}} */
-  def int(i: BigInt): TBuilderInstruction = unsafeBuilder.int(i).point[TBuilderInternalState]
+  def int(i: Int): TBuilderInstruction = unsafeBuilder.int(i).point[TBuilderInternalState]
```

Comment:
What is the reason for this change? We have put some effort to support big integers, instead of just integers. There is a practical need for that, as Cosmos and Ethereum contracts operate over big integers. If we use the typed builder to import specs in JSON (as we are planning to do), this interface will become a problem.

Reply:
This had more to do with generators and tests, where (`a until b`) didn't work with big integers and had to be cast. But you're right, I was thinking about how it's not a big deal since the builder was only ever called programmatically, but I suppose we need this if we want to use the builder in JSON importing. I'll change this back.